.env.example

```
@@ -1,12 +1,7 @@
-# Captcha provider: mock | googleRecaptchaEnterprise
-VITE_CAPTCHA_PROVIDER=googleRecaptchaEnterprise
+﻿# Client-side captcha settings
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The ﻿# Client-side captcha settings key has incorrect delimiter

(IncorrectDelimiter)